torch/cpp_build.py

```diff
@@ -0,0 +1,20 @@
+import inspect
+import os.path
+
+from setuptools.command.build_ext import build_ext
+
+
+class BuildExtension(build_ext):
+    """A custom build extension for adding compiler-specific options."""
+
+    def build_extensions(self):
+        for extension in self.extensions:
+            extension.extra_compile_args = ['-std=c++11']
+        build_ext.build_extensions(self)
+
+
+def include_paths():
```

Comment:
Currently all our code for extensions lives in `torch.utils.ffi`. We could put it there too